qvm/template-gui.jinja

```diff
@@ -45,7 +45,11 @@
         admin.vm.device.pci.Available       *   {{ vmname }}             dom0                             allow
         admin.vm.device.mic.Available       *   {{ vmname }}             dom0                             allow
         admin.vm.feature.Get        +internal   {{ vmname }}             dom0                             allow
-        admin.vm.feature.Get        +servicevm  {{ vmname }}             dom0                             allow
+        admin.vm.feature.Get       +servicevm  {{ vmname }}             dom0                             allow
+        admin.vm.feature.Get        +template   {{ vmname }}             @tag:guivm-{{ vmname }}          allow
+        # TODO: limit to templates related to @tag:guivm-{{ vmname }} only
+        admin.vm.feature.Get        +template   {{ vmname }}             @type:TemplateVM                 allow
+        admin.vm.feature.List               *   {{ vmname }}             @tag:guivm-{{ vmname }}          allow target=dom0
```

Comment:
This one shouldn't be needed, at least in theory. What specifically fails if you don't allow it?

Reply:
Can you try `qvm-appmenus --verbose --update dvm-myvm` in sys-gui-gpu to get more details about the failure (especially - what caused it to do that call)?

Reply:
hm, and no any actual error message? weird...